webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  entry: './src/index.tsx',
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'index_bundle.js',
```

Comment:
Also add `clean: true`. It's needed for clearing your dist folder before each build process, it's necessary to have an fresh files